kakarot_scripts/withdraw_accounts.py

```diff
@@ -0,0 +1,96 @@
+# %% Imports
+import logging
+import os
+from asyncio import run
+
+import requests
+
+from kakarot_scripts.constants import ETH_TOKEN_ADDRESS, NETWORK, RPC_CLIENT
+from kakarot_scripts.utils.starknet import (
+    get_balance,
+    get_declarations,
+    get_deployments,
+    invoke,
+)
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+# %% Fetch contract events
+def get_contracts():
+    contract_address = hex(get_deployments()["kakarot"]["address"])
+    logger.info(f"ℹ️  Fetching contracts from {contract_address}")
+    url = f"{NETWORK['voyager_api_url']}/events?ps=10&p=1&contract={contract_address}"
+    headers = {
+        "accept": "application/json",
+        "x-api-key": os.getenv("VOYAGER_API_KEY"),
+    }
+    response = requests.get(url, headers=headers)
+    return [
+        {
+            "evm_address": next(
+                item["value"]
+                for item in event["dataDecoded"]
+                if item["name"] == "evm_contract_address"
+            ),
+            "starknet_address": next(
+                item["value"]
+                for item in event["dataDecoded"]
+                if item["name"] == "starknet_contract_address"
+            ),
+        }
```

Comment:
I wanted to keep the names `evm_address` and `starknet_address` since I saw they are widely used in the repo, but I can change it if you think it’s better.

Reply:
Ok, fixed, I changed it a bit because using `event["dataDecoded"]` directly would have resulted in a structure like this, which wouldn’t have been good to use:

`[[{'name': 'evm_contract_address', 'value': '0xa77fda95c42813b974a4e81dbfd5b2f5bb0b108c', 'type': 'felt'}, {'name': 'starknet_contract_address', 'value': '0x36594e6aee5efab8c5e286f3d517e551aa07d24d133d4170df0a3582ee09ed0', 'type': 'felt'}], [{'name': 'evm_contract_address', 'value': '0x4b012de585e72d675d95da8d9b48174c936898ac', 'type': 'felt'}, {'name': 'starknet_contract_address', 'value': '0x7cdf16bac0d32fe06fe45bd9a67583467908907276f62e9d6b1904d689accca', 'type': 'felt'}]]`